static/src/javascripts/projects/common/modules/commercial/epic-utils.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 
+import fastdom from 'fastdom';
```

Comment:
I think `import fastdom from 'lib/fastdom-promise'` is required for `fastdom` to be "thenable".

Reply:
thanks - changes made last night without running frontend - checking it all now 👌

Reply:
No worries, I should've said in my original comment!